sys-fs/lvm2/lvm2-2.03.14-r4.ebuild

```diff
@@ -34,7 +35,7 @@ RDEPEND="${DEPEND_COMMON}
 	>=sys-apps/baselayout-2.2
```

Comment:
If you didn't already, can you audit the deps to make sure they're minimal for USE=-lvm?